webpack/sass/main.scss

```diff
@@ -1,4 +1,7 @@
 @import "../node_modules/@creativecommons/vocabulary/css/vocabulary.css";
+@import "../node_modules/@creativecommons/vocabulary/scss/spacing.scss";
+@import "../node_modules/@creativecommons/vocabulary/scss/color.scss";
+@import "../node_modules/@creativecommons/vocabulary/scss/typography.scss";
```

Comment:
This does not seem to be in use.

Reply:
This is to keep consistency with other PRs.

Reply:
And color tokens are used.